bindings/python/crocoddyl/core/diff-action-base.cpp

```diff
@@ -59,17 +59,17 @@ void exposeDifferentialActionAbstract() {
            "allocated. This function returns the allocated data for a predefined\n"
            "DAM.\n"
            ":return DAM data.")
-      .def("quasiStatic", &DifferentialActionModelAbstract_wrap::quasiStatic_x,
-           DifferentialActionModel_quasiStatic_wraps(
-               bp::args("self", "data", "x", "maxiter", "tol"),
-               "Compute the quasic-static control given a state.\n\n"
-               "It runs an iterative Newton step in order to compute the quasic-static regime\n"
-               "given a state configuration.\n"
-               ":param data: action data\n"
-               ":param x: discrete-time state vector\n"
-               ":param maxiter: maximum allowed number of iterations\n"
-               ":param tol: stopping tolerance criteria (default 1e-9)\n"
-               ":return u: quasic-static control"))
+      .def("quasiStatic", &DifferentialActionModelAbstract_wrap::quasiStatic,
+           &DifferentialActionModelAbstract_wrap::default_quasiStatic, bp::args("self", "data", "x", "maxiter", "tol"),
```

Comment:
Could you please correct the argument list corresponding to `quasiStatic` here?

Reply:
> Could you please correct the argument list corresponding to `quasiStatic` here?

done



>  Also, it seems that we are missing an overload for the case of `DifferentialActionModelAbstract::quasiStatic_x` now

Yes, you're right. I put it back.